### ISSUE TITLE
Can we launch onagui from command line? With which variables?

### ISSUE THREAD

**Issue by SteBiard** [lmazuel/onagui#85]
Hello,

Thanks a lot for this toolset.

Is there a way to interact programmatically with ONAGui? Is it doucmented somewhere?

The idea is to be able to load the 2 ontologies directly, launch one of the algorithm and be able to export the alignment as skos.

Thanks and regards

**Comment by tfrancart** [comment]
No, you cannot run the algorithms directly. That would be a nice addition, though.
Currently, there is AlignmentGUIRunner class that allows to run the GUI by preloading the 2 vocabularies, but you still need to run the alignment manually.
Also, the I_Sub comparison algorithm is available in [its own java class](https://github.com/lmazuel/onagui/blob/master/src/main/java/fr/onagui/alignment/method/I_Sub.java), it it helps.

**Comment by SteBiard** [comment]
Thanks a lot for the feedback and the hints to pre-load the vocabularies, it could likely help.
